.gitignore

```diff
@@ -3,3 +3,6 @@ dist
 node_modules
 .vscode-test/
 *.vsix
+
+snipsnap.code-snippets
+which-key-snippets.code-snippets
```

Comment:
Why do we need these?

Reply:
My bad, added these to make sure I didn't commit the actual files to this commit, will remove.